components/chat_widget/src/components/ocs-chat/ocs-chat.css

```diff
@@ -3,40 +3,23 @@
 @tailwind utilities;
 
 :host {
-   /**
-   * @prop --button-background-color: Button background color
-   * @prop --button-background-color-hover: Button background color on hover
-   * @prop --button-text-color: Button text color
-   * @prop --button-text-color-hover: Button text color on hover
-   * @prop --button-border-color: Button border color
-   * @prop --button-border-color-hover: Button border color on hover
```

Comment:
These should be restored since they are used by the docs generator to update the README file